README.md

```diff
@@ -63,11 +63,12 @@ jobs:
               tag: latest
           run:
             path: /usr/local/bin/pool_boy.sh
+          args:
```

Comment:
Comment 1:
I would prefer keeping the `params:` interface, because it is closer to how this script would behave if it where a Concourse resource instead of a simple task.

On the other hand, maybe this is not possible if we want to provide a separate timeout per tasks. Do you have an idea?

Comment 2:
no matter wether we can go back to the `params` interface or not, could you please augment the README with the 2 ways that this script can be used (that is 2 examples, 1 with a global timeout and 1 with multiple pools each with its own timeout). As-is, from the readme I don't understand how to use the script, I have to look at the code.